src/components/VisualTesterKeymap.vue

```diff
@@ -1,6 +1,17 @@
 <template>
   <div class="tester">
-    <div class="layout-selector-radios"></div>
```

Comment:
I know that I'm undoing your work, @yanfali, but in the absence of a ISO toggle in the settings panel, users should still be able to switch keyboard geometries without having to change the selected OS keyboard layout.

Reply:
Fair enough. I took it out because it seemed weird to not switch if you selected ISO in the settings panel. Now you've added locale it's more nuanced.